src/theme/icon.rs

```diff
@@ -302,55 +386,66 @@ impl IconTheme {
             ("gslides", "\u{f1c4}"),         // ""
             ("guardfile", "\u{e21e}"),       // ""
             ("gz", "\u{f410}"),              // ""
-            ("h", "\u{f0fd}"),               // ""
             ("hbs", "\u{e60f}"),             // ""
             ("heic", "\u{f1c5}"),            // ""
             ("heif", "\u{f1c5}"),            // ""
             ("heix", "\u{f1c5}"),            // ""
+            ("hh", "\u{f0fd}"),              // ""
             ("hpp", "\u{f0fd}"),             // ""
             ("hs", "\u{e777}"),              // ""
-            ("htm", "\u{f13b}"),             // ""
             ("html", "\u{f13b}"),            // ""
+            ("htm", "\u{f13b}"),             // ""
+            ("h", "\u{f0fd}"),               // ""
             ("hxx", "\u{f0fd}"),             // ""
             ("ico", "\u{f1c5}"),             // ""
             ("image", "\u{f1c5}"),           // ""
             ("img", "\u{f1c0}"),             // ""
             ("iml", "\u{e7b5}"),             // ""
+            ("info", "\u{e795}"),            // ""
             ("ini", "\u{e615}"),             // ""
             ("ipynb", "\u{e606}"),           // ""
             ("iso", "\u{f1c0}"),             // ""
+            ("j2", "\u{e000}"),              // ""
             ("jar", "\u{e738}"),             // ""
             ("java", "\u{e738}"),            // ""
+            ("jinja", "\u{e000}"),           // ""
+            ("jl", "\u{e624}"),              // ""
             ("jpeg", "\u{f1c5}"),            // ""
             ("jpg", "\u{f1c5}"),             // ""
-            ("js", "\u{e74e}"),              // ""
+            ("jsonc", "\u{e60b}"),           // ""
             ("json", "\u{e60b}"),            // ""
+            ("js", "\u{e74e}"),              // ""
             ("jsx", "\u{e7ba}"),             // ""
-            ("jl", "\u{e624}"),              // ""
             ("key", "\u{e60a}"),             // ""
             ("kt", "\u{e634}"),              // ""
             ("kts", "\u{e634}"),             // ""
             ("ksh", "\u{f489}"),             // ""
-            ("ld", "\u{e624}"),              // ""
+            ("kts", "\u{e634}"),             // ""
```

Comment:
Looks like this is repeated.

Reply:
> Looks like kt and kts is repeated.

won't it cause another merge conflict if i remove it

Reply:
No, it should be fine. The issues is mostly when you merge main into your branch. But don't bother about it now, we've enabled suash merge and I can do that from my end.